spec/grape_entity/hash_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe Grape::Entity::Hash do
+  let(:fresh_class) { Class.new(Grape::Entity) }
+
+  it 'Test' do
+    class Address < Grape::Entity
```

Comment:
These are global scope. Can you declare them with `let(:address) { Class.new(...) }` please?